### ISSUE TITLE
Failed to initialize quick mode

### ISSUE THREAD

**Issue by KrokusAstra** [ReplayMod/ReplayMod#1012]
Hi! I have this error. Yesterday i didn't have this problem, but now i have. The only thing i changes i think is fabric loader versions since i thought better version = more stability, and updated a couple of mods. I'm not sure, but i think i was on Fabric 0.16.5, and now i'm on 0.16.7. I tried to roll-back to 0.16.5, but it didn't helped.

Modloader - Fabric, version 0.16.7
Minecraft 1.20.1

![изображение_2024-10-30_185135390](https://github.com/user-attachments/assets/ad311a07-86df-46c3-a128-2c66c49640e2)

This is latest log file
https://mclo.gs/KF5EeTS

**Comment by Johni0702** [comment]
Please attach the affected replay.

**Comment by KrokusAstra** [comment]
Oh, sorry, i think i figured it out. I just deleted 2-3 mods which crashed my minecraft to fix it... and now even if world itself fine - block just disappeared, replays became broken. My bad

**Comment by KrokusAstra** [comment]
Nope, still same issue. I just recorded how i beat ender dragon, and travel through end-dimensions, didn't delete any mods, want to make video with replay mod, try to enter quick-mode, but got the error. I even record another 30s video to check, and no, there is no problem. But with 1.5h replay there is a problem. As if my PC spec just not enough to enter quick-mode fast enough and it reject me with error.
![изображение_2024-11-09_013607172](https://github.com/user-attachments/assets/44347435-9495-467a-9fc1-60a837376f83)


[Replay file](https://dropmefiles.com/1yR7y)

**Comment by KrokusAstra** [comment]
Also, i checked log file while starting a game, and log is full of this:
https://mclo.gs/I0nSY2X
